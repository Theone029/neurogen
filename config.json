{
  "exchange": "kraken",
  "api_key": "YOUR_KRAKEN_API_KEY_HERE",
  "api_secret": "YOUR_KRAKEN_SECRET_HERE",
  "trade_pair": "XBT/USD",
  "order_size": 0.01,
  "profit_target": 0.5,
  "stop_loss": 0.25,
  "poll_interval": 15,
  "log_level": "INFO"
}
