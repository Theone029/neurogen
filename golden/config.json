{
  "api_key": "YOUR_OPENAI_API_KEY_HERE",
  "dry_run": true,
  "exchange": {
    "pairlists": [
      {
        "method": "StaticPairList",
        "pairs": [
          "BTC/USD",
          "ETH/USD"
        ]
      }
    ],
    "name": "kraken",
    "key": "YOUR_KRAKEN_API_KEY_HERE",
    "secret": "YOUR_KRAKEN_SECRET_HERE",
    "ccxt_config": {},
    "ccxt_async_config": {},
    "pair_whitelist": [
      "BTC/USD",
      "ETH/USD"
    ],
    "pair_blacklist": []
  },
  "stake_currency": "USD",
  "stake_amount": "unlimited",
  "timeframe": "5m",
  "strategy": "NeuroStrike",
  "entry_pricing": {},
  "exit_pricing": {},
  "telegram": {
    "token": "PLACEHOLDER_TOKEN",
    "chat_id": "PLACEHOLDER_CHAT_ID",
    "enabled": false
  },
  "pairlists": [
    {
      "method": "StaticPairList",
      "pairs": [
        "BTC/USD",
        "ETH/USD"
      ]
    }
  ]
}
